k8s/namespaces/rd/rd-judicial-data-load-integration/rd-judicial-data-load-integration.yaml

```diff
@@ -14,4 +14,4 @@ spec:
   values:
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/judicial-data-load:prod-614cfd7-20210831113123
+      image: hmctspublic.azurecr.io/rd/judicial-data-load:prod-614cfd7-20210831113123   #{"$imagepolicy": "flux-system:rd-judicial-data-load-integration"}
```

Comment:
Can you change this policy to `rd-judicial-data-load`? It looks to be using the same, I would remove the image repo/policy for `rd-judicial-data-load-integration`

Reply:
Image policy/repo now updated.